recipes/c-client/all/cclient.def

```diff
@@ -0,0 +1,771 @@
+EXPORTS 
+	UW_copyright 	 DATA

```

Comment:
that's something hard to maintain in a long term.
but it's a makefile-based project, so there is no magical `CMAKE_WINDOWS_EXPORTS_ALL_SYMBOLS` :(

Reply:
It's unlikely that there will be changes upstream; that library is unmaintained for more than 10 years.